core/src/main/scala/org/apache/spark/TaskEndReason.scala

```diff
@@ -149,8 +149,8 @@ case class ExceptionFailure(
     this(e, accumUpdates, preserveCause = true)
   }
 
-  def exception: Option[Throwable] = exceptionWrapper.flatMap {
-    (w: ThrowableSerializationWrapper) => Option(w.exception)
+  def exception: Option[Throwable] = exceptionWrapper.flatMap { w =>
```

Comment:
Question: in this case, it could even be `flatMap(w => Option(w.exception))`. Is this different at all as far as Scala is concerned, without the braces at all? I could look it up but figured you might already know.